Bottom: 4d5f144bc80259d72eaae365361d0d81756afd87
Top:    de4b309f61c8988d128d3c468c9f1a32cb6a1e0d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 02:03:29 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
